Tests/Serialization/ResourceSerializerTests.cs

```diff
@@ -392,8 +411,8 @@ public void HandlesNullValues()
             Assert.NotNull(attributes["last-name"]);
             Assert.NotNull(attributes["age"]);
 
-            Assert.Null(relationships["job"]["data"]);
-            Assert.Null(relationships["friends"]["data"]);
+            Assert.Equal(0, relationships["job"]["data"].Count());
```

Comment:
`Assert.Null` in this context checks that the property isn't there at all? I think checking for count on a one-to-one relationship is confusing. In this case, if the relationship is `null`, I would expect the value to be `null`, so `relationships["job"]["data"].Type == JTokenType.Null`.

Reply:
Agreed, reworked to be more intuitive.